src/pages/components/number-input/usage.mdx

```diff
@@ -96,24 +131,177 @@ or decrease an incremental value.
 
 ## Formatting
 
-- Do not use number inputs when large value changes are expected. They work best
-  for making small, incremental changes that require only a few clicks.
-- Enable the user to also choose to type a number in the field.
-- Ensure that a default value is shown within the field.
-- Use a clear and concise label for the number input.
-- Use sentence-style capitalization (only the first word in a phrase and any
-  proper nouns capitalized) for the label.
-- See further guidance for using number labels within
-  [forms](/components/form/usage).
+### Anatomy
+
+<Row>
+<Column colLg={8}>
+
+![number input anatomy](images/number-input-usage-2.png)
+
+</Column>
+</Row>
+
+1. **Label**: Text that informs the user about the content they need to enter in
+   the field. It is required unless you get an approved accessibility exemption.
+2. **Numeric value**: The value changes when users entered into the field.
+3. **Helper text** (optional, except for error and warning): Assistive text that
```

Comment:
Error and warning messages aren't helper text but they do replace helper text when they appear.

Reply:
Oh, I see. I did the same for progress bar. Do you think it's best to call them out separately from the helper text? Or we can just put it in the same line with Helper text but explain that they replace the helper text?

Reply:
I think putting them in the same line is fine.